.gitignore

```diff
@@ -1,3 +1,14 @@
+*.iml
```

Comment:
I understand your rationale for separation, it sounds good. But is that some standard we try to adhere or a common practice for some ecosystem that we want to adopt?

What I meant was - I do not understand, why is this bad to have few extra lines (that do not require attention/maintenance) but do save some time for some people (who use particular IDE and do not use global .gitignore).

Personally, I do not see a good reason for spending time changing this and then spending more time setting up global .gitignore without strong arguments (i.e avoid something bad, increase something good, etc).

At the same time, please do let me know if you or anybody else have strong opinions or arguments about that.